### DISCUSSION TITLE
Always change kubernetes.repo in centos7

### DISCUSSION THREAD

**Discussion by shadowofmoo** [kubermatic/kubeone#3344] [Q&A] [answered]
I have a better mirror about kubernetes, when use kubeone it always change the kubernetes mirror

**Comment by embik** [comment, marked as answer]
Hi @shadowofmoo, you can set the following in your `kubeone.yaml` to avoid KubeOne configuring repositories:

```yaml
systemPackages:
  configureRepositories: false
``` 

Also see [our reference](https://docs.kubermatic.com/kubeone/v1.8/references/kubeone-cluster-v1beta2/#kubeonecluster) for all available settings.
